presto-spi/src/main/java/com/facebook/presto/spi/security/PrincipalType.java

```diff
@@ -11,7 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.facebook.presto.hive.metastore;
+package com.facebook.presto.spi.security;
```

Comment:
I am not sure if it is a good idea. Connector may have different principal types than Presto. Maybe it is ok for now.